### PR TITLE
Implement tests for commands associated with Room

### LINKED ISSUE #69: Code Suggestion: Move getTypicalAddressBook to its own class
## How things are
Currently, `getTypicalAddressBook` is located in `seedu.address.testutil.TypicalResidents`.

## Suggestion
We should consider moving this to its own class

## Reason
An address book is a different thing from a resident

### PR DESCRIPTION
**Pull request by DrWala**
### What this does:
Increases code coverage for commands associated with the room class
Coincidentally closes #69 

### How to test:
1. Checkout the project
2. Run tests for the project
3. Ensure all tests are passing

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/room/EditRoomCommandTest.java [hunk 1]
```diff
@@ -1,4 +1,186 @@
...
+import seedu.address.model.UserPrefs;
+import seedu.address.model.room.Room;
+import seedu.address.testutil.room.EditRoomDescriptorBuilder;
+import seedu.address.testutil.room.RoomBuilder;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for EditRoomCommand.
+ */
 public class EditRoomCommandTest {
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
```
Comment (stein414) [inline src/test/java/seedu/address/logic/commands/room/EditRoomCommandTest.java:34]: The tests cases seems extensive. Good work

**Review by stein414 — APPROVED** [review]
LGTM

File: src/test/java/seedu/address/testutil/TypicalAddressBook.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package seedu.address.testutil;
```
Comment (benedictkhoomw) [inline src/test/java/seedu/address/testutil/TypicalAddressBook.java:1]: Thank you for moving this to its own class. Could you update your PR to close #69?

**Review by benedictkhoomw — APPROVED** [review]
Cloned, tested and skimmed code. LGTM.

**Pull request merged**